spec/features/embargo_h2_spec.rb

```diff
@@ -12,10 +12,10 @@
 RSpec.describe 'H2 item created with embargo; change embargo in Argo',
                type: :feature, stage_only: true, embargo: true, skip_for_default: true do
   let(:collection_title) { RandomWord.nouns.next }
-  let(:item_title) { "Text file for #{collection_title}" }
+  let(:item_title) { "Text file #{RandomWord.nouns.next} in H2 collection" }
   let(:user_email) { "#{AuthenticationHelpers.username}@stanford.edu" }
-  let(:purl_xml_embargo_prefix) { '<access type="read">\s*<machine>\s*<embargoReleaseDate>' }
-  let(:purl_xml_embargo_suffix) { '</embargoReleaseDate>\s*<none.*>\s*</machine>\s*</access>' }
+  let(:purl_xml_embargo_prefix) { '<access type="read">\s*<machine>\s*<none.*>\s*<embargoReleaseDate>' }
+  let(:purl_xml_embargo_suffix) { '</embargoReleaseDate>\s*</machine>\s*</access>' }
```

Comment:
I'm not in love with caring about whether the embargo element is first or last here.   Ideas?   I'm not looking to deal with the entire purl.xml

Reply:
xpath query?

Reply:
so slurp up the entire purl and treat it as xml?   could do.